src/Lachain.Console/Program.cs

```diff
@@ -99,6 +100,9 @@ private static void UpdateDb(DbOptions options)
                 if(options.depth <= 0) throw new ArgumentException("depth must be positive integer");
                 System.Console.WriteLine("Starting hard db optimization");
 
+                LogManager.Configuration.Variables["consoleLogLevel"] = "Trace";
```

Comment:
Try to read it from environment before using the hardcoded value (see example here - https://github.com/LATOKEN/lachain/blob/31833d8d3800cabbac3a642dbfac6d6ab77293dd/src/Lachain.Console/Application.cs#L48). And check this 2 logs initializations do not conflict in main run regime please

Reply:
Yes, tried it. But 'DbOptions' do not have any LogLevel. Perhaps I will add LogLevel in DbOptions and then do it like this.

Reply:
Sorry did not notice the last line. On it.

Reply:
No,  change it with environments only,  you don't need it in options really in this case

Reply:
so,  with environment you can start it as 
```
LOG_LEVEL=Warning ./Lachain.Console db ...
```

Reply:
Sorry,  I missed you have implemented it already,  it is ok now